backend/dataall/base/db/exceptions.py

```diff
@@ -147,6 +147,18 @@ def __str__(self):
         return f'{self.message}'
 
 
+class AWSResourceQuotaExceeded(Exception):
```

Comment:
nit: `ServiceQuota` instead of `ResourceQuota` to match the actual name on AWS

Reply:
I was thinking of this file containing sort of generic exception ( but user defined ), and so I named this as ResourceQuota and it could applied to anywhere quota is exceeded. Do you still prefer to name it with ServiceQuota

Reply:
I would go for ServiceQuota, because of the AWS prefix. We can make it generic when we need it